CodeMaid/Helpers/CodeItemTypeComparer.cs

```diff
@@ -123,6 +138,20 @@ private static int CalculateConstantOffset(BaseCodeItem codeItem)
             return codeItemField.IsConstant ? 0 : 1;
         }
 
+        private static int CalculateExplicitInterfaceOffset(BaseCodeItem codeItem)
```

Comment:
Another nitpick, but can you move this above CalculateConstantOffset so these methods appear in the order they are invoked?